### PR TITLE
First pass at refactoring to match Steve's designs

### PR DESCRIPTION
**Pull request by kimberleejohnson**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,56 +1,44 @@
...
-    .on('participant-updated', updateParticipantInfoDisplay)
-    .on('participant-left', updateParticipantInfoDisplay)
-    .on('left-meeting', hideCallDisplay);
+    .on('left-meeting', toggleCallPanel);
 
-  let roomURL = document.getElementById('room-url');
-  const joinButton = document.getElementsByClassName('join-call')[0];
+  const roomURL = document.getElementById('url-input');
+  const joinButton = document.getElementsByClassName('start-button')[1];
+  const createButton = document.getElementsByClassName('start-button')[0];
```
Comment (philmillman) [inline index.js:25]: I think `getElementById` will be easier instead of all these `getElementsByClassName`

File: index.js [hunk 2]
```diff
@@ -82,114 +70,85 @@ async function createRoom() {
...
-// Changes the color of the 'join' button once a room has been created
-async function createDemoRoom() {
-  const createButton = document.getElementById('create-button');
-  const joinButton = document.getElementsByClassName('join-call')[0];
-  const roomURL = document.getElementById('room-url');
-  createButton.innerHTML = 'Creating room...';
+async function createRoomAndStart() {
+  const createAndStartButton = document.getElementsByClassName(
+    'start-button'
+  )[0];
```
Comment (philmillman) [inline index.js:79]: same as above re: id

**Review by philmillman — CHANGES_REQUESTED** [review]
As discussed, I think switching most of the `getElementsByClassName` to `getElementById` is best. 

Looks good otherwise. Great job!

**Comment by kimberleejohnson** [comment]
> As discussed, I think switching most of the `getElementsByClassName` to `getElementById` is best.
> 
> Looks good otherwise. Great job!

Thank you, @philmillman! All those `getElementsByClassName` are `getElementById`'s now, and the header issue should be resolved as well (no more scroll). Re-requesting review!

**Review by philmillman — APPROVED** [review]

**Pull request merged**